src/sql.ts

```diff
@@ -1,35 +1,36 @@
 import { Plugin, PluginAttachmentDB, PluginConfig, PluginConfigId, PluginError, PluginsServer } from './types'
 
+function pluginConfigsInForceQuery(specificField?: string): string {
```

Comment:
Suggestion: replace `string` with `'id' | 'plugin_id'`. This makes the following string substitution a lot less scary from an injection standpoint.

Reply:
Good idea, I used `keyof PluginConfig` for a bit of flexibility